src/main/java/org/jenkinsci/test/acceptance/update_center/LocalOverrideUpdateCenterMetadataDecoratorImpl.java

```diff
@@ -61,32 +61,31 @@ public void decorate(UpdateCenterMetadata ucm) {
         }
         for (Map.Entry<String,String> e : System.getenv().entrySet()) {
             String key = e.getKey();
-            String name;
-            if (key.endsWith(".jpi")) {
-                name = key.replace(".jpi", "");
-            } else if (key.endsWith("_JPI")) { // http://stackoverflow.com/a/36992531/12916
-                name = null;
-                for (String _name : ucm.plugins.keySet()) {
-                    if ((_name.toUpperCase(Locale.ENGLISH).replaceAll("[^A-Z0-9_]", "_") + "_JPI").equals(key)) {
```

Comment:
So what is now happening that allows this to be deleted? Does `LocalOverride.create` reliably infer the name? In that case there is no reason to set distinct environment variables at all. You could simply define

    LOCAL_OVERRIDES=/path/to/whatever.jpi:/another/path/to/something-else-1.33-irrelevant-filename.jpi

Reply:
Yes, the name is inferred from the plugin by looking at its manifest.

Reply:
Added a variable `LOCAL_JARS` to do this